### ISSUE TITLE
Directory for log files is not created in HTML5 export

### ISSUE THREAD

**Issue by kaiec** [godotengine/godot#47472]
**Godot version:** 3.2.3

**OS/device including version:** Linux

**Issue description:** In HTML5 export, the log file is not written if the directory does not exist. In other exports it gets created.

**Steps to reproduce:** Configure log files at a directory under user://, e.g. using the default value user://logs/godot.log. Export to HTML5 and open the log file within Godot. You get error 7, File does not exist. It works for user://godot.log, i.e. without a directory. It also works at least under Linux, where the log directory simply gets created if it does not exist.

**Minimal reproduction project:**
[bugreport.zip](https://github.com/godotengine/godot/files/6224413/bugreport.zip)

**Comment by Calinou** [comment]
HTML5 doesn't have a physical filesystem you can access on the OS. Instead, it saves data to localStorage. This makes logs difficult to access for casual users, and therefore not very practical.

**Comment by kaiec** [comment]
Yes, and this is why I expose the log file in-game in the GUI so that casual users can see it and can send it to me. It works perfectly, also under HTML5, but only, if the log file is written to user:// directly, not to a subdirectory. I can live with that but finding this out took me quite some time that I would like to spare people coming after me.

**Comment by Calinou** [comment]
cc @Faless
